validator/coordinator/src/main/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandler.java

```diff
@@ -134,14 +135,21 @@ public ValidatorApiHandler(
   }
 
   @Override
-  public SafeFuture<Optional<List<ValidatorDuties>>> getAttestationDuties(
+  public SafeFuture<Optional<List<AttesterDuties>>> getAttestationDuties(
       final UInt64 epoch, final Collection<Integer> validatorIndexes) {
     if (isSyncActive()) {
       return NodeSyncingException.failedFuture();
     }
     if (validatorIndexes.isEmpty()) {
       return SafeFuture.completedFuture(Optional.of(emptyList()));
     }
+    if (epoch.isGreaterThan(combinedChainDataClient.getCurrentEpoch().plus(UInt64.ONE))) {
```

Comment:
```suggestion
    if (epoch.isGreaterThan(combinedChainDataClient.getCurrentEpoch().plus(Constants.MIN_SEED_LOOKAHEAD))) {
```

is probably more accurate.  We should do better at using that constant rather than hard coding 1 all the time...

Reply:
i had never associated that variable with this, its more of a max_lookahead i would have been looking for than that name...